build_tools/teatime.py

```diff
@@ -95,6 +176,25 @@ def finish(self):
                     f"END\t{end_time}\t{end_time - self.start_time}\n".encode()
                 )
             self.log_file.close()
+
+            # Call log indexing before upload
+            amdgpu_families = os.getenv("AMDGPU_FAMILIES", "unknown")
+            indexer_script = Path("build_tools/index_logs.sh")
+            indexer_args = [
+                str(indexer_script),
+                amdgpu_families,
+                str(self.log_path.parent),
+                "build/indexer.py",
+            ]
+            if indexer_script.exists():
+                try:
+                    print(f"[teatime] Running {indexer_script} for {amdgpu_families}")
+                    subprocess.run(indexer_args, check=True)
```

Comment:
The bash script will always fail as you the script `build_tools/index_logs.sh` get downloads at a later step in the workflow the way the workflow is structured right now.